plugins/woocommerce/client/blocks/assets/js/base/stores/woocommerce/cart.ts

```diff
@@ -324,6 +328,24 @@ const { state, actions } = store< Store >(
 
 					// Dispatches the event to sync the @wordpress/data store.
 					emitSyncEvent( { quantityChanges } );
+
+					const errors = [];
+					for ( const response of json.responses ) {
+						const body = response?.body;
+						if (
+							body &&
+							typeof body === 'object' &&
+							'code' in body &&
+							'message' in body
+						) {
+							errors.push( body );
```

Comment:
We could call `actions.showNoticeError()` directly here without creating an `errors` array, no?